Source/WebCore/platform/graphics/mac/controls/SwitchThumbMac.mm

```diff
@@ -58,7 +58,7 @@
 {
     LocalDefaultSystemAppearance localAppearance(style.states.contains(ControlStyle::State::DarkAppearance), style.accentColor);
 
-    bool isOn = style.states.contains(ControlStyle::State::Checked);
```

Comment:
I just realized this means we can simplify RenderTheme::isChecked. (Alternatively we could make that take on this logic, but this seems cleaner? Not sure.)